redis/src/commands/mod.rs

```diff
@@ -2229,8 +2229,15 @@ impl ToRedisArgs for ScanOptions {
         }
     }
 
-    fn is_single_arg(&self) -> bool {
-        false
+    fn num_of_args(&self) -> usize {
```

Comment:
minor: I believe that `len` is the idiomatic choice

Reply:
The Vec and Slice types have the `len` method, so I think it'd be better to separate them to make it clear. How do you think?